data/Luminometrics/dataintegrity.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Nov 23 16:40:14 2016
+
+@author: Vivian Zhong
+"""
```

Comment:
Requires a bit more documentation. Make clear that `compiler.py` has to be run before `dataintegrity.py`. Alternatively, may be good to throw and error that states that `hash.log` is not present and that `compiler.py` needs to be run first.